src/workers/indexMarkdown.ts

```diff
@@ -10,20 +10,16 @@ const indexMarkdown = new DiscordWorker(
     const childrenValues = await job.getChildrenEntries()
     const { markdown }: { markdown: string } = childrenValues
 
-    const chunkSize = 1000
-    const overlapSize = 200
-
-    const chunks: string[] = []
-    for (let i = 0; i < markdown.length; i += chunkSize - overlapSize) {
-      const chunk = markdown.slice(i, i + chunkSize).trim()
-      if (chunk.length > 0) chunks.push(chunk)
-    }
+    const chunks = markdown
+      .split('\n###')
+      .map((p) => p.trim())
```

Comment:
```suggestion
      .map((p) => '\n###' + p.trim())
```
vi slänger bort en radbrytning så det behöver vi lägga tillbaka